recipe/meta.yaml

```
@@ -19,7 +19,9 @@ source:
     - fix_blas_lapack.patch
 
 build:
-  number: 1
+  number: 2
+  run_exports:
+    - {{ pin_subpackage('pytorch', max_pin='x.x') }}
```

As per https://github.com/conda-forge/pytorch-cpu-feedstock/issues/40#issuecomment-803133327, shouldn't this be 

```suggestion
    - {{ pin_subpackage('pytorch', max_pin='x.x.x') }}
```